ROADMAP.md

```diff
@@ -511,26 +511,29 @@ advances made by Mozilla's
 Not fully supporting the browser platform will hurt most of the projects betting
 on libp2p.
 
-### 🌡️ Opt-in telemetry protocol
+### 🌡️ Telemetry protocol
 
-**What?** By developing a *decentralized* telemetry protocol that nodes can opt
-into, we could build a world-wide real-time visualisation of our global libp2p
+**What?** By developing a *privacy-preserving* telemetry protocol,
```

Comment:
I agree telemetry is important. 
Should we split the telemetry protocol effort and metrics support effort into two separate items?  or change the title to be both the protocol and metrics?

Reply:
> Should we split the telemetry protocol effort and metrics support effort into two separate items?

Aren't they already? The metrics effort is described in the "Full Observability" section.

Reply:
I see, sorry my miss. Thanks for clarifying!  The "collect stats" statement made me misunderstood this item.